pkg/apis/serving/metadata_validation_test.go

```diff
@@ -147,11 +150,62 @@ func TestValidateObjectMetadata(t *testing.T) {
 				"testAnnotation": "testValue",
 			},
 		},
+	}, {
+		name: "revision initial scale not parseable",
```

Comment:
Shouldn't all the negative test live close to the implementation, i.e. in the autoscaling validation?

Reply:
I agree. Should we remove these negative test cases here then?